makePoplogCommander.sh

```diff
@@ -257,7 +257,10 @@ static_assert( false, "Not defined for operating systems other than Darwin nor L
 
 cat << \****
 
-const char * const USEPOP = "[//USEPOP//]";
+#define USEPOP_LITERAL "[//USEPOP//]"
```

Comment:
What's the reason for this? Is it so that you can do string-concatenation at compile time? I guess `USEPOP "/auto"` doesn't work on line 383? Is there much benefit to having the `const char * USEPOP` in addition to `USEPOP_LOCAL`? I'd imagine the compiler will coalesce all the occurences of the string into a single location in read only storage?

Reply:
Yes, purely to avoid string concatenation at runtime. If you can do it at compile-time you know it is safe and, of course, you avoid the runtime overhead.